frontend/components/execs.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "../styles/team.module.css";
+import Image from "next/image";
+import Link from "next/link";
+
+const Execs = (props) => {
+  const socials = {
+    LinkedIn: "https://www.linkedin.com/in/" + props.linkedin + "/",
+    Facebook: "https://www.facebook.com/" + props.fb,
+    email: "mailto:" + props.email,
+  };
+
+  return (
+    <div className={styles.execCol}>
+      <div className={styles.profile}>
+        <div>
+          {props.imgUrl && (
+            <div className={styles.profileCrop}>
+              <div className={styles.profileImg}>
+              <Image
+                src={props.imgUrl}
+                className={`${styles[props.className]}`}
+                alt={props.name}
+                width="250px"
+                height="250px"
+              />
+              </div>
+            </div>
+          )}
+          <div className={styles.profileDetails}>
+            <p className={styles.profileTextName}>{props.name}</p>
+            <p className={styles.profileTextPosition}>{props.position}</p>
+            {props.degree && (
+              <p className={styles.profileDegree}>
+                {props.degree}
```

Comment:
Add more padding for degree body text and fix icon dimensions. It should match what is on the website currently. 

Submitted: 
![image](https://user-images.githubusercontent.com/80564580/174698096-1a97ac54-f88e-4904-8665-fb3754981af9.png)

On the website: 
![image](https://user-images.githubusercontent.com/80564580/174698118-4c061a1d-cedf-40d2-af72-08eb22cd9955.png)